tck/src/main/java/org/eclipse/microprofile/lra/tck/TckTests.java

```diff
@@ -45,398 +36,508 @@
+    public void getActiveLRAs() throws WebApplicationException {
+        URL lra = lraClient.startLRA(null, lraClientId(), lraTimeout(), ChronoUnit.MILLIS);
 
-        assertNotNull(getLra(lras, lra.toExternalForm()), "getActiveLRAs: getLra returned null", null);
+        List<LRAInfo> allActiveLRAs = lraManagement.getLRAs(LRAStatus.Active);
+        boolean isLraIdInList = containsLraId(allActiveLRAs, lra);
 
-        lraClient.closeLRA(lra);
+        assertTrue("LRA '" + lra + "' should be listed in the list of the active LRAs, as it was not closed yet, but it isn't",
+                isLraIdInList);
 
-        return lra.toExternalForm();
+        lraClient.closeLRA(lra);
     }
 
+    /**
+     * Started LRA should be listed amongst active LRAs.
+     */
     @Test
-    private String getAllLRAs() throws WebApplicationException {
-        URL lra = lraClient.startLRA(null, "SpecTest#getAllLRAs", LRA_TIMEOUT_MILLIS, ChronoUnit.MILLIS);
-        List<LRAInfo> lras = lraSPI.getLRAs(null);
+    public void getAllLRAs() throws WebApplicationException {
+        URL lra = lraClient.startLRA(null, lraClientId(), lraTimeout(), ChronoUnit.MILLIS);
 
-        assertNotNull(getLra(lras, lra.toExternalForm()), "getAllLRAs: getLra returned null", null);
+        List<LRAInfo> allClientLRAs = lraManagement.getLRAs(null);
+        boolean isLraIdInList = containsLraId(allClientLRAs, lra);
 
-        lraClient.closeLRA(lra);
+        assertTrue("LRA '" + lra + "' should be listed in the list of the all LRAs, as it was not closed yet, but it isn't",
+                isLraIdInList);
 
-        return PASSED_TEXT;
+        lraClient.closeLRA(lra);
     }
 
-    //    @Test
-    private void getRecoveringLRAs() throws WebApplicationException {
+    @Test
+    @Ignore
+    public void getRecoveringLRAs() throws WebApplicationException {
         // TODO
     }
 
+    /**
+     * Started LRA should in state 'active'.
+     */
     @Test
-    private String isActiveLRA() throws WebApplicationException {
-        URL lra = lraClient.startLRA(null, "SpecTest#isActiveLRA", LRA_TIMEOUT_MILLIS, ChronoUnit.MILLIS);
+    public void isActiveLRA() throws WebApplicationException {
+        URL lra = lraClient.startLRA(null, lraClientId(), lraTimeout(), ChronoUnit.MILLIS);
 
-        assertTrue(lraSPI.getStatus(lra).isActive(), null, null, lra);
+        assertTrue("LRA '" + lra + "' is not denoted as active even it was started", lraManagement.getStatus(lra).isActive());
 
         lraClient.closeLRA(lra);
-
-        return lra.toExternalForm();
     }
 
-    // the coordinator cleans up when canceled
+    /**
+     * Canceled LRA should in state 'compensated'.
+     * NOTE: the coordinator cleans up when canceled
+     */
     @Test
-    private String isCompensatedLRA() throws WebApplicationException {
-        URL lra = lraClient.startLRA(null, "SpecTest#isCompensatedLRA", LRA_TIMEOUT_MILLIS, ChronoUnit.MILLIS);
+    @Ignore
```

Comment:
I forget why I disabled some tests but the ones you have marked @Ignore will need adding to issue #92

Reply:
+1

yes, the tests marked as `@Ignore` are not run in current TCK. Even they were not marked as Ignore they were rejected from running by not being defined in the list of permitted tests.
Are you fine that this step of checking what are purposes of tests will be run as separate follow-up task after this PR will be merged and when SmallRye client implementation is merged to repo (for being possible to run the TCK tests - e.g. those ignored ones - with some existing implementation)?